job_templates/packaging_job.xml.em

```diff
@@ -152,8 +154,11 @@ if [ -z "${CI_ROS2_REPOS_URL+x}" ]; then
   CI_ROS2_REPOS_URL="@default_repos_url"
 fi
 export CI_ARGS="$CI_ARGS --repo-file-url $CI_ROS2_REPOS_URL"
-if [ "$CI_TEST_BRIDGE" = "true" ]; then
-  export CI_ARGS="$CI_ARGS --test-bridge"
+if [ -n "${CI_ROS2_SUPPLEMENTAL_REPOS_URL+x}" ]; then
+  export CI_ARGS="$CI_ARGS --supplemental-repo-file-url $CI_ROS2_SUPPLEMENTAL_REPOS_URL"
+fi
```

Comment:
Left over from previous iterations?

Reply:
I'd need that as long as e.g. `rosbags2` are not the `ros2.repos` file.

Reply:
Currently this variable is not exposed anywhere in the Jenkins UI so you don't have a way to set it and the Python script doesn't have an argument `--supplemental-repo-file-url`. So as is it is unused code.

And you don't need that option. You can simply pass a custom `.repos` file including `rosbag2` through the existing option `CI_ROS2_REPOS_URL`.

Reply:
I'll expose it through the jenkins ui then. Having a custom repos file only for rosbag doesn't make sense to me as I would have to constantly keep track of the original repos file and update my custom one accordingly.

Reply:
> Having a custom repos file only for rosbag doesn't make sense to me as I would have to constantly keep track of the original repos file and update my custom one accordingly.

Why would a custom repos file be necessary in the first place? With these changes can't you simple add `rosbag2` to the `ros2.repos` file - the same as `ros1_bridge`? I thought that was the goal of this - to always test `rosbag2` as any other package.

Reply:
That would be desired but for this we would have to build the rosbridge for every CI. Which we could once https://github.com/ros2/ros1_bridge/pull/164 landed.

Reply:
It could still be limited to the packaging jobs - the same as it is right now. Just with `rosbag2` in addition to `ros1_bridge`.